### DISCUSSION TITLE
Helmsman

### DISCUSSION THREAD

**Discussion by jihoon-seo** [cloud-barista/cb-ladybug#81] [General]
https://github.com/Praqma/helmsman

> Helmsman is a Helm Charts (k8s applications) as Code tool which allows you to automate the deployment/management of your Helm charts from version controlled code.

**Comment by jihoon-seo** [comment]
Helmfile
https://github.com/roboll/helmfile

> Helmfile is a declarative spec for deploying helm charts. It lets you...
> 
> * Keep a directory of chart value files and maintain changes in version control.
> * Apply CI/CD to configuration changes.
> * Periodically sync to avoid skew in environments.
> 
> To avoid upgrades for each iteration of `helm`, the `helmfile` executable delegates to `helm` - as a result, `helm` must be installed.

**Comment by sykim-etri** [comment]
두 프로젝트을 비교한 링크입니다. 현재는 helmfile이 contributor 수 등에서 크게 앞서고 있습니다.
https://github.com/roboll/helmfile/issues/240
https://github.com/Praqma/helmsman/issues/18

helmfile은 golang의 특징을 매우 잘 사용하고 있는 것 같아서 따라가기가 너무 어려운 단점이 있습니다.
